src/meta.ml

```diff
@@ -257,14 +257,23 @@ let builtins ~stdlib_dir =
         ]
     }
   in
+  let dynlink =
```

Comment:
Shouldn't this be just `let dynlink = simple "dynlink" ~dir:"+" []` like e.g. `bigarray` above ?

Reply:
Indeed. Thanks.